dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -10,7 +10,7 @@
   - script_data = { editorall: { video_infos: video_infos }.to_json }
   %script{src: minifiable_asset_path('js/levels/editors/_all.js'), data: script_data}
 
-- unless @level.is_a? DSLDefined
+- unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
```

Comment:
This file is a mess 😛.  Long-term we need to come up with a better solution for deciding which editor partial to show for which level model.  For now though I'm not sure it makes sense for `Unplugged` and `DSLDefined` to both inherit from a common sub-class.